File: challenges/5-ordenar-letras/go/joao-erik2077/ordenarletras.go
package main

import (
	"fmt"
	"os"
	"sort"
	"strings"
)

func main() {
	word := os.Args[1]
	orderned_word := sort_word(word)
	fmt.Println(orderned_word)
}

func sort_word(word string) (sorted_word string) {
	word_to_array := strings.Split(word, "")
	sort.Strings(word_to_array)
	sorted_word = strings.Join(word_to_array, "")
	return
}

File: challenges/5-ordenar-letras/java/lukasveiga/OrdenarLetras.java
import java.util.Arrays;
import java.util.Scanner;

public class OrdenarLetras {

    public static void main(String[] args) {

        try (var scan = new Scanner(System.in)){

            System.out.println("Informe uma palavra: ");
            var word = scan.nextLine();

            System.out.println(orderLetters(word));
            
        } catch (Exception e) {
            System.out.println(e.getMessage());
        }
        
    }

    private static String orderLetters(String word) {
        var letters = word.split("");
        Arrays.sort(letters);
        return String.join("", letters);
    }
}
